sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/service/json/JsonConverterTest.java

```diff
@@ -0,0 +1,696 @@
+package io.stargate.sgv2.docsapi.api.service.json;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.BooleanNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.stargate.proto.QueryOuterClass.ColumnSpec;
+import io.stargate.proto.QueryOuterClass.Row;
+import io.stargate.proto.QueryOuterClass.Value;
+import io.stargate.sgv2.docsapi.api.common.properties.document.impl.DocumentPropertiesImpl;
+import io.stargate.sgv2.docsapi.config.DocumentConfig;
+import io.stargate.sgv2.docsapi.config.constants.Constants;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class JsonConverterTest {
```

Comment:
could this be a `@QuarkusTest` where you simply inject the `JsonConverter` and actually test with respect to the setup data store and document properties, created object mapper, etc.. You can also adapt properties if needed, for example `stargate.document.max-dept=6`. You can see other tests to figure out how the setup a dedicated config for the test?